### PR TITLE
Edits for ease of testing

### PR DESCRIPTION
**Pull request by ialarmedalien**
This PR does not change the functionality, but puts in place some changes to make future work easier.

- Creating a new Docker image for ease of testing checkM, with the prerequisite binaries and python modules already installed. Saved to `ialarmedalien/kbase_checkm_base:latest`. `Dockerfile_checkM` creates this base image, and `Dockerfile` adds the local python directory on top. `Dockerfile` from the previous version has been renamed `Dockerfile_full`. Once the app updates and testing are complete, I will move the files back again.
- Adding self to `kbase.yml` author list.
- Switched the current assembly and genome queries to use smaller datasets to speed up tests.
- Ran json pretty print over the UI spec files since there was a mix of tabs and spaces ( :o ) and the keys were not ordered.

### CODE REVIEW

File: ui/narrative/methods/run_checkM_lineage_wf/spec.json [hunk 1]
```diff
@@ -1,125 +1,146 @@
...
-        "dylan","msneddon","dparks","tgu2","seanjungbluth"
-    ],
-    "contact": "http://kbase.us/contact-us/",
-    "categories": ["active","communities","assembly"],
-    "widgets": {
-        "input": null,
-        "output": "no-display"
-    },
-    "parameters": [ 
+  "authors": [
```
Comment (ialarmedalien) [inline ui/narrative/methods/run_checkM_lineage_wf/spec.json:2]: Added myself to author list, pretty-printed JSON.

File: ui/narrative/methods/run_checkM_lineage_wf_withFilter/spec.json [hunk 2]
```diff
@@ -1,178 +1,213 @@
...
-        "dylan","msneddon","dparks","tgu2","seanjungbluth"
-    ],
-    "contact": "http://kbase.us/contact-us/",
-    "categories": ["active","communities","assembly"],
-    "widgets": {
-        "input": null,
-        "output": "no-display"
-    },
-    "parameters": [ 
+  "authors": [
```
Comment (ialarmedalien) [inline ui/narrative/methods/run_checkM_lineage_wf_withFilter/spec.json:2]: Added myself to author list, pretty-printed JSON.

File: Dockerfile_checkM [hunk 3]
```diff
@@ -0,0 +1,66 @@
+FROM kbase/sdkbase2:python
```
Comment (ialarmedalien) [inline Dockerfile_checkM:1]: This loads the prerequisites (checkM, python modules). I've pushed the resulting image to dockerhub -- it speeds up development a lot if you don't have to wait for the docker image to build locally / on GitHub / on the kbase servers.

File: test/core_checkM_test.py [hunk 4]
```diff
@@ -241,7 +241,7 @@ def prep_assemblies(self):
                         'ref':   getattr(self, a['attr']),
                         'label': a['name'],
                     }
-                    for a in assembly_list
+                    for a in assembly_list[0:3]
```
Comment (ialarmedalien) [inline test/core_checkM_test.py:244]: these files are much smaller ==> speed up the tests!

**Comment by dcchivian** [comment]
unit tests are failing, all with a message of the form

File "/miniconda/lib/python3.6/shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: '/kb/module/work/tmp/test_data/assemblies/GCF_002817975.1_ASM281797v1_genomic.fna'

**Comment by ialarmedalien** [comment]
@dcchivian how are you running the tests?

**Pull request merged**